README.md

```diff
@@ -36,11 +36,20 @@ Via `homebrew`:
 $ brew tap twilio/brew && brew install twilio
 ```
 
+## Link the Twilio CLI Infra Plugin
+
+To get started, run `npm install` to install the plugin package dependencies. Afterwards, run the following command:
+
+```sh-session
+$ twilio plugins:link <path_to_your_local_repo>
```

Comment:
@bermudezmt the `plugins:link` command is only needed to test local changes. When we merge all the fix to `main`, user will be able to use:
```
twilio plugins:install kaiquelupo/plugin-twilio-infra
```
or
```
twilio plugins:install twilio-infra-as-code/plugin-twilio-infra
```
 to install the plugin. 

The way the CLI works is that it uses `yarn` to install the plugin package. `yarn` is picking up the package either from npmjs or from github.

Reply:
You mean once this becomes an official npm package, right? Or are you actually able to run `twilio plugins:install` for this package now?

Reply:
You are able to do it already @bermudezmt without publishing the package on npm. The CLI uses `yarn` to fetch the plugin, and `yarn` can use github directly to fetch and install a new depenendecy. Just keep in mind that yarn is going to fetch the `main` branch of the repo.

Reply:
Thank you. I've updated.